edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/MessageHelper.cs

```diff
@@ -14,5 +15,25 @@ public static string GetSenderId(this IMessage message)
 
             return string.Empty;
         }
+
+        public static string GetOutput(this IMessage message)
```

Comment:
Nit: GetOutput() => TryGetValue() ? outputName : string.Empty;

Reply:
Does this improve readability? I find the of statement faster to parse visually